packages/components/tests/e2e/specs/vsCodeExtension.spec.js

```diff
@@ -706,33 +706,30 @@ context('VS Code Extension', () => {
     it('highlight and loop through events selected from Trace view filter', () => {
       cy.get('.tabs .tab-btn').last().click();
 
-      cy.get('.trace-view__search-input-suffix').should('not.exist');
-      cy.get('.trace-view__search-input-element').type('example');
-      cy.get('.trace-view__search-input-element').should(
-        'have.value',
-        'example'
-      );
-      cy.get('.trace-view__search-input-suffix').click();
-      cy.get('.trace-view__search-input-element').should('have.value', '');
+      cy.get('.trace-filter__suffix').should('not.exist');
+      cy.get('.trace-filter__input').type('example');
+      cy.get('.trace-filter__input').should('have.value', 'example');
+      cy.get('.trace-filter__suffix').click();
+      cy.get('.trace-filter__input').should('have.value', '');
 
       let eventQuery = 'id:1 label:json id:3 id:15 id:99999 #link_to_edit_url';
```

Comment:
Also worth testing:
- Hitting ESC clears focus from the text input and the suggestion list is hidden
- After submitting the form, the suggestion list is still visible
- Two suggestions are selected and the text input contains the expected string
- If we allow quoted literals containing whitespace, verify they return the expected results in a case where they'd otherwise return unexpected results if each word was treated as its own query (`"SELECT COUNT"` vs `SELECT COUNT`)
- Search terms are case-insensitive

Reply:
Added new e2e test for filter suggestions https://github.com/applandinc/appmap-js/pull/534/commits/ab513bf07e98b30335eb29f80bcc490490ab1d94